packages/block-library/src/embed/styles.native.scss

```diff
@@ -72,6 +72,24 @@
 	align-items: center;
 }
 
+.embed-preview__wrapper {
+	flex: 1;
+}
+
+.embed-preview__wrapper--align-left {
+	align-items: flex-start;
+}
+.embed-preview__wrapper--align-right {
+	align-items: flex-end;
+}
+
+.embed-preview__sandbox--align-left {
+	max-width: 360px;
+}
+.embed-preview__sandbox--align-right {
+	max-width: 360px;
+}
```

Comment:
The `max-width` value is based on the styles from the web version:
https://github.com/WordPress/gutenberg/blob/253d9b6e2123a2806b8843562f3f37de186c933a/packages/block-library/src/embed/style.scss#L7